common/ellipses.js

```diff
@@ -42,6 +43,7 @@
                         else if (scope.config.editable)
                             editLink = scope.tuple.reference.contextualize.entryEdit.appLink;
 
+
                         if (editLink) {
                             scope.edit = function () {
```

Comment:
[Unlink](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L61) should also be wrapped.